<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="can_interface" default="can0" />
  <arg name="can_bitrate" default="500000" />
  <arg name="control_rate" default="100.0" />
  <arg name="use_sim" default="false" />

  <!-- Dexter Hardware Bridge Node -->
  <node pkg="dexter_hardware" exec="dexter_hardware_bridge.py" name="dexter_hardware_bridge" output="screen">
    <param name="can_interface" value="$(var can_interface)" />
    <param name="can_bitrate" value="$(var can_bitrate)" />
    <param name="control_rate" value="$(var control_rate)" />
    <param name="use_sim" value="$(var use_sim)" />
  </node>

</launch>
